oak-core/src/main/java/org/apache/jackrabbit/oak/plugins/index/IndexUpdate.java

```diff
@@ -72,6 +73,16 @@
 
     private static final Logger log = LoggerFactory.getLogger(IndexUpdate.class);
     private static final String TYPE_ELASTICSEARCH = "elasticsearch";
+    public static final String DEFAULT_indexJcrTypeInvalidLogLimiter = "1000";
+    //This is used so that wrong index definitions are sparsely logged. After every 1000 indexing cycles, index definitions
+    // with wrong nodetype will be logged.
+    private static final long indexJcrTypeInvalidLogLimiter = Long.valueOf(System.getProperty("oak.indexer.indexJcrTypeInvalidLogLimiter", DEFAULT_indexJcrTypeInvalidLogLimiter));
```

Comment:
This should be all caps. And then I think it can be public, so you can use it from the test.

Reply:
I am not setting this in test using system property as it is "static final" --> this gets initialised first with default value whenever any test runs and then I am unable to change it in the test. I don't want to add a setter or not make it final just to pass the test. 

Will change it to all caps

Reply:
Making it public will keep it read-only (as it's final). I thought making it public so it can be read in the test.